examples/acl/variables.tf

```diff
@@ -0,0 +1,46 @@
+#####################################################
+# Network ACL Module Example Parameters
+# Copyright 2020 IBM
+#####################################################
+
+variable "create_vpc" {
+  description = "True to create new VPC. False if VPC is already existing and subnets or address prefixies are to be added"
+  type        = bool
+}
+
+variable "enable_acl" {
+  description = "Enables acl id if true"
+  type        = bool
+  default     = true
+}
+
+variable "name" {
+  description = "Name of the Network ACL"
+  type        = string
+}
+
+variable "vpc_name" {
+  description = "VPC name"
+  type        = string
+}
+
+#####################################################
+# Optional Parameters
+#####################################################
+
+variable "resource_group" {
+  description = "Resource group name"
+  type        = string
+  default     = null
+}
+
+variable "rules" {
```

Comment:
Define the contents of rules variable

variable "rules" {
  description = "List of rules that are to be attached to the Network ACL"
  type = list(object({
    name        = string
    action      = string
    source      = string
    destination = string
    direction   = string
    icmp = object({
      code = number
      type = number
    })
    tcp = object({
      port_max        = number
      port_min        = number
      source_port_max = number
      source_port_min = number
    })
    udp = object({
      port_max        = number
      port_min        = number
      source_port_max = number
      source_port_min = number
    })
  }))
  default = []
}

Reply:
adding readme block for the object type should suffice